squad/ci/models.py

```diff
@@ -20,11 +20,6 @@ def list_backends():
 class Backend(models.Model):
     name = models.CharField(max_length=128, unique=True)
     url = models.URLField()
-    # listener_url is used by backend's listen() method
-    listener_url = models.URLField(null=True, blank=True)
```

Comment:
I wouldn't go that far. LAVA offers a discovery method. Other services might not. Of course the problem is just 'theoretical' right now as we don't support other services for listening.